include/modem/nrf_modem_lib_trace.h

```diff
@@ -69,5 +69,10 @@ int nrf_modem_lib_trace_process(const uint8_t *data, uint32_t len);
  */
 int nrf_modem_lib_trace_stop(void);
 
+/**
+ * @brief Deinitialize trace module.
+ */
+void nrf_modem_lib_trace_deinit(void);
```

Comment:
Perhaps I forgot something. But why was this added? Dont we need an entry in the change log for this? Need unit tests for this (can be done in a separate PR)

Reply:
This was added because transports need to be de-initialized just like they need to be initialized.
Once the modem is shutdown, there must be a way for the application to de-initialize these as well.

Reply:
In particular in case the modem has crashed and the modem library needs to re-initialized.

Reply:
I see. Thanks. How about an entry in the change log for this?